core/modules/moduleconf.py

```diff
@@ -38,11 +60,12 @@ def canDelete(self, skel: SkeletonInstance) -> bool:
 
     @classmethod
     def get_by_module_name(cls, module_name: str):
```

Comment:
```suggestion
    def get_by_module_name(cls, module_name: str) -> Union[None, SkeletonInstance]:
```
We can add a Typehinte here
and import it in the top  ` from typing import  Union`